Doc/c-api/arg.rst

```diff
@@ -59,9 +59,9 @@ which disallows mutable objects such as :class:`bytearray`.
    the length argument (int or :c:type:`Py_ssize_t`) is controlled by
    defining the macro :c:macro:`PY_SSIZE_T_CLEAN` before including
    :file:`Python.h`.  If the macro was defined, length is a
-   :c:type:`Py_ssize_t` rather than an :c:type:`int`. This behavior will change
+   :c:type:`Py_ssize_t` rather than an :c:expr:`int`. This behavior will change
```

Comment:
Could it be possible to add the definition of the `:c:type:` role in the CPython specific extension?

Reply:
The ``:c:type:`` is still there, but it is (and was) meant for cross-referencing declared types, e.g., here for ``Py_ssize_t`` is is fine. Fundamental types like ``int`` and pointer types (e.g., ``Py_ssize_t``)  can not be declared, so they can not be cross-referenced. They can still be typeset with relevant links, which is what the ``:c:expr:`` role is intended for.

Reply:
Thank you for your clarification @jakobandersen. I hate to merge such large changes, but seems this is the only correct solution. Great work!